index.js

```diff
@@ -140,9 +140,9 @@ class ProtomuxRpcClient extends ReadyResource {
   async _makeRequest (methodName, args, { requestEncoding, responseEncoding }) {
     if (this.opened === false) await this.opening
 
-    while (!this.rpc && !this.closing) {
+    while ((!this.rpc || this.rpc.closed) && !this.closing) {
```

Comment:
Before this, users would get a channel_closed error if sending a request after coming back from suspending (because rpc is defined, but has been closed)